dts/bindings/clock/st,stm32n6-cpu-clock-mux.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2022, Linaro ltd
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  STM32N6 CPU Clock
+  Describes the STM32N6 CPU clock multiplexer.
+  For instance:
+  &perck {
+    clocks = <&rcc STM32_SRC_HSI_KER CKPER_SEL(0)>;
+    status = "okay";
+  };
+
+compatible: "st,stm32n6-cpu-clock-mux"
+
+include:
+  - name: base.yaml
+    property-allowlist:
+      - status
+      - compatible
+      - clocks
+
+properties:
+  clock-frequency:
+    required: true
+    type: int
+    description: |
+      default frequency in Hz for clock output
```

Comment:
`      Default frequency in Hz for clock output`. 
Also, be more explicit on the actual clock this represents (sys_cpu_ck)

Reply:
Done